pkgs/desktops/gnome-3/extensions/pop-os-shell/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, nodePackages, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-os-shell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev =  "${version}";
+    sha256 = "sha256-gv9knhG1/HOUsIyPWfBRBpJWg/bfRXbbg/x2QDgSmls=";
+  };
+
+  buildInputs = [ glib nodePackages.typescript ];
+
+
+  uuid = "pop-shell@system76.com";
+
+  makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
+
+  meta = with stdenv.lib; {
+    description = "Keyboard-driven layer for GNOME Shell which allows for quick and sensible navigation and management of windows";
```

Preference for <80 characters for description. If you would like to have an extended description, please use `longDescription`
```suggestion
    description = "Keyboard-driven layer for GNOME Shell";
```